internal/pkg/routes/route.go

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Mainflux
+// SPDX-License-Identifier: Apache-2.0
+
+package routes
+
+import (
+	"fmt"
+
+	mqtt "github.com/eclipse/paho.mqtt.golang"
+	"github.com/mainflux/mainflux/logger"
+	"github.com/nats-io/nats.go"
+)
+
+// Route - message route, tells which nats topic messages goes to which mqtt topic.
+// Later we can add direction and other combination like ( nats-nats).
+// Route is used in mfx and plain. Route is like base implementation and mfx and plain
+// are extended implemntation.
+type Route struct {
+	NatsTopic string
+	MqttTopic string
+	Subtopic  string
+	Logger    logger.Logger
+	Mqtt      mqtt.Client
+}
+
+type RouteIF interface {
+	Consume(m *nats.Msg)
+	Publish([]byte)
+}
+
+func NewRoute(n, m, s string, mqtt mqtt.Client, l logger.Logger) RouteIF {
+	return &Route{
+		NatsTopic: n,
+		MqttTopic: m,
+		Subtopic:  s,
+		Mqtt:      mqtt,
+		Logger:    l,
+	}
+}
+
+func (r Route) Consume(m *nats.Msg) {
+	r.Publish(m.Data)
```

Comment:
Why are we publishing in the `Consume()` function? This does not make too much sense (it is confusing) - looks like the name is poorly chosen. Should not `Forward` or `Transfer` be better?